src/ORM/Rule/ExistsIn.php

```diff
@@ -98,6 +109,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        if ($this->_options['passingOnPartialNulls'] === true
```

Comment:
The option is a boolean and return value of the method is also boolean. So can just do:
`if ($this->_options['passingOnPartialNulls'] && $this->_checkPartialSchemaNulls($entity, $source))`